code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm

```diff
@@ -8,6 +8,9 @@
 	req_one_access = list(ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_PREP, ACCESS_MARINE_CARGO)
 	hackable = TRUE
 
+/obj/structure/machinery/cm_vending/sorted/cargo_guns/squad_prep/ui_state(mob/user)
+	return GLOB.not_incapacitated_and_adjacent_strict_state
```

Comment:
same here

Reply:
As discussed on discord - this is to allow for req vendors to be non-strict but squad prep vendors to be strict